src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -323,6 +372,9 @@ public virtual bool TryHandleBackTab()
                 {
                     ExpansionSession.EndCurrentExpansion(fLeaveCaret: 1);
                     ExpansionSession = null;
+                    _symbols = default;
+                    _symbol = null;
+                    _arguments = _arguments.Clear();
                 }
```

Comment:
don't love the large duplication with this and above.

Reply:
➡️ Tracking state containment as https://github.com/dotnet/roslyn/pull/49916#discussion_r544816693